src/FilamentSpatieLaravelActivitylogServiceProvider.php

```diff
@@ -22,7 +22,7 @@ public function configurePackage(Package $package): void
     protected function getResources(): array
     {
         return [
-            ActivityResource::class,
+            config('filament-spatie-laravel-activitylog.resource.filament-resource') ?? ActivityResource::class,
```

Comment:
What do we think about adding the `ActivityResource::class` to the config and simplifying this to just reference the config since you have it there already?

Reply:
I did originally have that in but when I upgraded the package in composer first time it errored. This was because the new variable was missing from the config on the update alone so I added this as work around. I think it may only update the config on peoples projects if they publish again after the update.

Reply:
Happy to take your lead on this and it may have just been my composer environment

Reply:
I'll need a couple of days to get back to you, off for a long weekend without my laptop but I think that's a good call on still including it within the service provider so that its not a breaking change.